lib/widgets/pokemon_tile_item.dart

```diff
@@ -19,37 +27,22 @@ class PokemonTileItem extends StatelessWidget {
         ),
         child: Column(
           children: [
-            const Text(
-              str.nameFiller,
-              style: TextStyle(
+            Text(
+              pokemon.name.capitalize(),
+              style: const TextStyle(
                 fontSize: 20,
                 color: Colors.black,
               ),
             ),
             Row(
               children: [
-                Column(
-                  children: const [
-                    Text(
-                      str.typeFiller,
-                      style: TextStyle(
-                        fontSize: 10,
-                        color: Colors.black,
-                      ),
-                    ),
-                    Text(
-                      str.typeFiller,
-                      style: TextStyle(
-                        fontSize: 10,
-                        color: Colors.black,
-                      ),
-                    ),
-                  ],
+                /// TODO: this widget will be later improved on https://feilfeilfeil.atlassian.net/browse/PKM-32
+                Image.network(
+                  pokemon.url.getImage(),
```

Comment:
have this ready instead before it's passed so the Image widget will immediately render it's component instead of waiting its String transformation to get done